docs/source/conf.py

```diff
@@ -171,3 +197,15 @@
 
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = True
+
+# -- Nitpick Ignore options --------------------------------------------------
+
+# Nitpick requires all references to be resolved
+# This will ignore those that references that can't be linked
+nitpick_ignore = []
```

Comment:
Not sure why you through this in, but if it's complaining about things not being linkable (and they link locally for you) try adding `sphinx==v7.2.6` to `requirements.txt`. I've had to do this in both Chemist and SimDE.